SparkPost/2web2ui
# SA-1622 - (UI) Compare By - Remove feature flag

[SA-1622](https://sparkpost.atlassian.net/browse/SA-1622)

### What Changed

- Removes references to the `allow_compare_by` feature flag from the app

### How To Test

1. Remove the `allow_compare_by` account UI option from your account
2. Verify the compare by UI still renders on `/signals/analytics` (with Hibana enabled)

### To Do
